apps/librespeed/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.8"
+services:
+  librespeed:
+    image: lscr.io/linuxserver/librespeed:5.4.1
+    container_name: librespeed
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=${TZ}
+      - PASSWORD=${APP_PASSWORD}
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/config
```

Comment:
It would be better so skip the extra config directory to avoid permission errors.